modules/110-istio/images/cni-v1x21x6/Dockerfile

```diff
@@ -1,8 +1,8 @@
-# Based on https://github.com/istio/istio/blob/1.16.2/cni/deployments/kubernetes/Dockerfile.install-cni
-ARG BASE_DISTROLESS
-FROM docker.io/istio/install-cni:1.16.2@sha256:14255494cf4c78740913c702aa0768fad4b433e82d10cdfc264ef5f30cf041ec as artifact
+# Based on https://github.com/istio/istio/blob/1.21.6/cni/deployments/kubernetes/Dockerfile.install-cni
+ARG BASE_ALT
```

Comment:
Why did you change `BASE_DISTROLESS` to `BASE_ALT` here and elsewhere?
`BASE_DISTROLESS' is the preferred option.

Reply:
Return Distroless in operators werf manifests. In other image need research for migrate to distroless.